runtime/libpgmath/lib/common/dmod.c

```diff
@@ -7,15 +7,10 @@
 
 #include "mthdecls.h"
 
-#ifndef WIN64
+#ifndef TARGET_WIN_X8664
 double
 __mth_i_dmod(double f, double g)
 {
-/* Need to do this way until a bug in the Win64 fmod routine is fixed */
```

Comment:
Was this deadcode?

Reply:
As @bryanpkc pointed it out earlier this was never compiled due to the outer `#ifndef WIN64` .